ui/assets/icons/search.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
```

Comment:
No more makeStyles.

Please use styled components. (MUI v5 compatible)

Reply:
The thing all the Hoc and parent components are styles with v4 styles thats why when v5 compatible styles they either gets override or have conflicts and don't work